backend/api/tests/integration/image/recent.rs

```diff
@@ -128,17 +128,19 @@ async fn update() -> anyhow::Result<()> {
 
     let client = reqwest::Client::new();
 
+    // updating
     let resp = client
```

Comment:
Since this endpoint now has a response, we should assert the response snapshot after